src/main/java/net/mcreator/minecraft/MinecraftImageGenerator.java

```diff
@@ -554,21 +554,42 @@ public static BufferedImage generateAchievementPreviewPicture(Workspace workspac
 		}
 
 		/**
-		 * <p>This method generates command images.</p>
+		 * <p>This method generates the basic command image preview.</p>
 		 *
 		 * @param command <p>The command.</p>
 		 * @return <p>Returns generated image.</p>
 		 */
-		public static BufferedImage generateCommandPreviewPicture(String command) {
+		public static BufferedImage generateBasicCommandPreviewPicture(String command, String procedurexml) {
```

Comment:
Why is this called _basic preview_, is there gonna be some kind of _advanced preview_ later?

Reply:
The basic preview is the current preview (`/commandName`), while the advanced preview is the procedure style preview.